packaging/macOS/parsec.spec

```diff
@@ -1,6 +1,7 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
 
 import os
+exec(open("../../parsec/_version.py", encoding="utf8").read())
```

Comment:
should be written like https://github.com/Scille/parsec-cloud/blob/25025075426176385d8b1adb0b1f563071ee6fa8/setup.py#L10-L12 to explain why